recipes/tcsh/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "tcsh" %}
+{% set version = "6.20.00" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tcsh-org/tcsh.git
+  git_tag: TCSH6_20_00
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - ncurses
+    - gcc
+  run:
+    - ncurses
+
+test:
+  imports:
+    - tcsh --version
+
+about:
+  home: http://www.tcsh.org
+  license: BSD
```

Comment:
Please include version of `BSD` (e.g. `BSD 3-Clause` for the 3-Clause variant of BSD).